pandas/_libs/window/aggregations.pyx

```diff
@@ -302,7 +302,7 @@ cdef inline float64_t calc_var(int64_t minp, int ddof, float64_t nobs,
             result = ssqdm_x / (nobs - <float64_t>ddof)
             # Fix for numerical imprecision.
             # Can be result < 0 once Kahan Summation is implemented
-            if result < 1e-15:
+            if result < 1e-14:
```

Comment:
xref https://github.com/pandas-dev/pandas/issues/37051, needed to change for an expanding groupby test